FoodAndBeer.md

```diff
@@ -0,0 +1,7 @@
+# Minneapolis/St. Paul, MN
+
+### Places to Eat
+- Dark Horse Brewery :beer - Downtown St. Paul
```

Comment:
@Kyle-Schiemo Is this place any good.  I think the user would like an explanation of **why** Dark Horse Brewery would be a good recommendation. i.e.  Is the :beer: good?